### PR TITLE
Silence some warnings in safari, improve error rendering on standalone

### PR DESCRIPTION
**Pull request by kainino0x**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,12 @@
 <html>
   <head>
     <title></title>
+    <link
+      id="favicon"
+      rel="shortcut icon"
+      type="image/png"
+      href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQAgMAAABinRfyAAAACVBMVEUAAAAAAAD///+D3c/SAAAAAXRSTlMAQObYZgAAAEpJREFUCB0FwbERgDAMA0BdSkbJQBSuaPABE0WuaKILmpJ/rNVejPKBUXGhqAC5J0gn9ESg2wvdNua8hUoKJQo8b6HyE6a2QHdbP0CPITh2pewWAAAAAElFTkSuQmCC"
```
Comment (beaufortfrancois) [inline index.html]: It is so cute ;)

**Comment by beaufortfrancois** [comment]
I'm not sure which code changes actually silence warnings in safari in this PR.

File: src/framework/fixture.ts [hunk 2]
```diff
@@ -66,12 +66,12 @@ export class Fixture {
 
   private expectErrorValue(expectedName: string, ex: unknown, m: string): void {
     if (!(ex instanceof Error)) {
-      this.fail('THREW NON-ERROR');
+      this.fail('THREW non-error value, of type ' + typeof ex);
```
Comment (kainino0x) [inline src/framework/fixture.ts]: This isn't silencing a warning, but it verbosely explains a specific error in Safari.

File: index.html [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
+    />
```
Comment (kainino0x) [inline index.html]: Silences 404 error on favicon.ico (should also silence a warning in chrome)

File: third_party/normalize.min.css [hunk 4]
```diff
@@ -1,2 +1 @@
...
-/*# sourceMappingURL=normalize.min.css.map */
```
Comment (kainino0x) [inline third_party/normalize.min.css]: Silences 404 on normalize.min.css.map

**Pull request merged**